bin/citrea/provers/risc0/guest-bitcoin/Cargo.lock

```diff
@@ -2919,6 +2919,7 @@ dependencies = [
  "serde",
  "sha2",
  "thiserror",
+ "tracing",
```

Comment:
Wow

Reply:
The dependency is optional and would only be used in native mode

Reply:
then it shouldn't have ended up in the bitcoin prover cargo.lock